stable/consul/values.yaml

```diff
@@ -68,6 +68,13 @@ uiService:
   enabled: true
   type: "NodePort"
 
+## Create an Ingress for the Web UI
+uiIngress:
```

Comment:
I personally prefer the pattern from `chartutil` when you `helm create foo`:
```yaml
ingress:
  enabled: false
  # Used to create an Ingress record.
  hosts:
    - chart-example.local
  annotations:
    # kubernetes.io/ingress.class: nginx
    # kubernetes.io/tls-acme: "true"
  tls:
    # Secrets must be manually created in the namespace.
    # - secretName: chart-example-tls
    #   hosts:
    #     - chart-example.local
```

Where working, example values are commented out and explained, rather than the user needing to remove a `[]` or `{}` in their override values file. But again I don't think this is standardized for charts yet either (for example, `Resources` above uses a mix of these patterns, just as `chartutil` currently does.

This is mainly for maintainer discussion - I don't see it as a blocker for this PR.